app/views/govuk_component/docs/task_list.yml

```diff
@@ -18,6 +18,7 @@ accessibility_criteria: |
   - allow users to show or hide all steps at once
   - inform the user which step a link belongs to
   - inform the user which step the current page is in, particularly where there are multiple steps in one group
+  - be readable when only the text of the page is zoomed - achieved for the numbers and logic elements by their text being wrapped in several elements that give them a white background and ensure the text when zoomed expands to the left, not to the right, where it would obscure the step titles
```

Comment:
Did you mention that this only happens in Firefox? Is it worth explicitly stating this so people don't remove the code, test in a different browser and assume it all still works fine?

Reply:
It happens in Firefox because Firefox has the option to only zoom the text, not the page. Restricting the definition to the action rather than the browser means we won't have to keep updating this line if other browsers add this functionality (or if it turns out they already have it, I don't know. They might also be able to do it with an extension, for example).